editor/editor_data.cpp

```diff
@@ -534,6 +535,7 @@ Variant EditorData::instantiate_custom_type(const String &p_type, const String &
 				if (n) {
 					n->set_name(p_type);
 				}
+				n->set_meta(CoreStringNames::get_singleton()->__custom_type_script, script);
```

Comment:
```suggestion
				n->set_meta(CoreStringName(__custom_type_script), script);
```
New preferred way of using core stringnames.